lib/marionette/element.js

```diff
@@ -306,9 +306,11 @@
      */
     size: function size(callback) {
       var cmd = {
-        name: 'getElementSize'
+        name: 'getElementRect'
       };
-      return this._sendCommand(cmd, 'value', callback);
+      var resp = this._sendCommand(cmd, 'value', callback);
+      console.log(resp);
+      return resp;
```

Comment:
It depends if we're fine with returning an object with two extra fields or not.  `getElementRect` returns a data structure that looks like

```json
{
  x: <number>,
  y: <number>,
  height: <number>,
  width: <number>
}
```

that is compatible with any client code.

What are your feelings on this?